calculator.html

```diff
@@ -6,47 +6,36 @@
 </head>
 
 <body>
-    <div class="calculator-container">
-        <div class="display">
-            <span id="top-display-text"></span>
-            <span id="bottom-display-text"></span>
+    <div class="calculator">
+        <div class="calculator__screen">
+            <span class="calculator__screen-top" id="calculator-screen-top"></span>
+            <span class="calculator__screen-bottom" id="calculator-screen-bottom"></span>
         </div>
-        <div class="big-btn-div"> 
-            <div class="logo-div">
-                <span id="top-logo-text">mawulator 1.0</span>
-                <span id="bottom-logo-text">i.../ᐠ｡ꞈ｡ᐟ\</span> 
+        <div class="calculator__logo-btn-wrapper"> 
```

Comment:
If I am not wrong, you are calling this `calculator__logo-btn-wrapper` because it contains a logo and a button. It's not a bad class name but it wouldn't be very scalable because whenever you add a new element to this element you will have to rename the class name 😯 

A suggestion I could give here is since this element looks like a toolbar to me, I would call this `calculator__toolbar` instead 🙃